component/media/js/jcomments-v4.0.js

```diff
@@ -1473,63 +1483,56 @@ JComments.prototype = {
 };
 
 document.addEventListener('DOMContentLoaded', function () {
-	document.body.addEventListener('click',function(e) {
+	document.body.addEventListener('click', async function (e) {
 		if (e.target && e.target.id == 'addcomments') {
 			e.preventDefault();
-
 			var el = e.target;
-
 			jcomments.showForm(el.dataset.object_id, el.dataset.object_group, 'comments-form-link');
 		} else if (e.target && e.target.id == 'comments-form-send') {
 			e.preventDefault();
-
-			jQuery.when(jcomments.saveComment()).done(function() {
-                var jc = window.jcomments;
-                if (jc) {
-                    setTimeout(() => {
-                        jc.showPage(jc.oi, jc.og, 0);
-                    }, 200);
-                }
-            });
+			try {
+				await jcomments.saveCommentAsync();
+				var jc = window.jcomments;
+				if (jc) {
+					setTimeout(() => {
+						jc.showPage(jc.oi, jc.og, 0);
+					}, 200);
+				}
+			} catch (error) {
+				console.error('Error saving comment:', error);
+			}
 		} else if (e.target && e.target.id == 'comments-form-reset') {
 			e.preventDefault();
 
 			document.querySelector('#comments-form').reset();
 		} else if (e.target && (e.target.id == 'comments-form-captcha-image' || e.target.id == 'cmd-captcha-reload')) {
 			e.preventDefault();
-
 			jcomments.clear('captcha');
 		}
 	});
-})
-
-jQuery(document).ready(function ($) {
-	$('.cmd-subscribe').on('click',function (e) {
-		e.preventDefault();
-
-		let $this = $(this);
-
-		Joomla.request({
-			url: $this.attr('href') + '&format=json',
-			onSuccess: function (response) {
-				let _response = JSON.parse(response);
-
-				if (_response.success) {
-					$this.attr({
-						href: _response.data.href,
-						title: _response.data.title
-					});
-					$this.html('<span aria-hidden="true" class="icon-mail icon-fw"></span> ' + _response.data.title);
 
-					Joomla.renderMessages({'message': [_response.message]}, '.comments-list-footer');
-				} else {
-					Joomla.renderMessages({'warning': [_response.message]}, '.comments-list-footer');
+	document.querySelectorAll('.cmd-subscribe').forEach(function (subscribeButton) {
```

Comment:
Only one link to subscribe in document. Use querySelector() instead.

Reply:
Fixed!